src/bower/Configuration.js

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
This file shouldn't be here right now. It was refactored and deleted. Did you resolve the merge conflicts?